### PR TITLE
Add documentation for lifx.effect_sky

### PR DESCRIPTION
**Pull request by Djelibeybi**
## Proposed change

Add documentation for the new `lifx.effect_sky` effect for LIFX Ceiling devices.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/121820
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Introduced the Sky effect for LIFX devices with attributes for sky types like Sunrise, Sunset, and Clouds, along with customizable palettes and settings.
  - Expanded descriptions for Flame, Morph, Sky, and Move effects to include new device compatibility and operational details.
  
- **Enhancements**
  - Added new attributes to the `lifx.effect_sky` service, including `entity_id`, `speed`, `palette`, `power_on`, `sky_type`, `cloud_saturation_min`, and `cloud_saturation_max` for more tailored control of lighting effects.

### CODE REVIEW

File: source/_integrations/lifx.markdown [hunk 1]
```diff
@@ -187,6 +187,51 @@ The device will be powered on by default, but this can be overridden by setting
 | `theme` | The theme to use for the effect. Must be one of: `autumn`, `blissful`, `cheerful`, `dream`, `energizing`, `epic`, `exciting`, `focusing`, `halloween`, `hanukkah`, `holly`, `independence` `day`, `intense`, `mellow`, `peaceful`, `powerful`, `relaxing`, `santa`, `serene`, `soothing`, `sports`, `spring`, `tranquil`, `warming`.
 | `power_on` | Whether to turn the light on before starting the effect (optional, default: true)
 
+### Service `lifx.effect_sky`
+
+Run a hardware-based effect on LIFX Ceiling devices that animates a sky scene across the device. There are three different types of sky emulated by the effect: Sunrise, Sunset and Clouds.
```
[the remark this answers is not in the document]
Comment (Djelibeybi) [reply source/_integrations/lifx.markdown]: LIFX Ceiling is a product name and thus "Ceiling" in this context must be capitalised.

File: source/_integrations/lifx.markdown [hunk 2]
```diff
@@ -122,7 +122,7 @@ script:
 
 ### Hardware effects
 
-The Flame (`lifx.effect_flame`), Morph (`lifx.effect_morph`) and Move (`lifx.effect_move`) effects are hardware-based and only work on specific LIFX devices. Flame and Morph are available on the LIFX Tile and Candle, while the Move effect requires a LIFX Z, Lightstrip, or Beam.
+The Flame (`lifx.effect_flame`), Morph (`lifx.effect_morph`), Sky (`lifx.effect_sky`) and Move (`lifx.effect_move`) effects are hardware-based and only work on specific LIFX devices. Flame and Morph are available on the LIFX Tile, Candle, Path, Spot and Ceiling while the Sky effect is only available on the Ceiling. The Move effect requires a LIFX Z, Lightstrip, Beam, Neon or String.
```
[the remark this answers is not in the document]
Comment (Djelibeybi) [reply source/_integrations/lifx.markdown]: These are all product names, thus the capital letter is correct in this context.

File: source/_integrations/lifx.markdown [hunk 3]
```diff
@@ -187,6 +187,51 @@ The device will be powered on by default, but this can be overridden by setting
...
+
+| Service data attribute | Description                                                                         |
+| ---------------------- | ----------------------------------------------------------------------------------- |
+| `entity_id`            | String or list of strings containing the `entity_id` of one or more Ceiling devices |
+| `speed` | Duration in seconds for the effect to complete (optional, min: 1s, max: 86400s, default: 50s)      |
+| `palette` | A list of 6 colors to use for this effect (optional, see below for details)                      |
+| `power_on` | Whether to turn the light on before starting the effect (optional, default: true)               |
+| `sky_type` | Either "Sunrise", "Sunset" or "Clouds"                                                          |
+| `cloud_saturation_min` | The minimum cloud saturation for the Cloud sky type (optional, default: 50)         |
+| `cloud_saturation_max` | The maximum cloud saturation for the Cloud sky type (optional, default: 180)        |
```
[the remark this answers is not in the document]
Comment (Djelibeybi) [reply source/_integrations/lifx.markdown:203]: The markdown is fine.

File: source/_integrations/lifx.markdown (hunk 1 above)
Comment (bdraco) [inline source/_integrations/lifx.markdown]:
```suggestion
Run a hardware-based effect on LIFX Ceiling devices that animates a sky scene across the device. The effect emulates three different types of sky: Sunrise, Sunset, and Clouds.
```

File: source/_integrations/lifx.markdown [hunk 4]
```diff
@@ -187,6 +187,51 @@ The device will be powered on by default, but this can be overridden by setting
...
+
+The palette for the Sky effect is shared between all three sky types. To use a custom palette, you must specify all six color values as hue (0-360), saturation (0-100), brightness (0-100) and kelvin (1500-9000) in the following order to modify the effect:
```
Comment (bdraco) [inline source/_integrations/lifx.markdown]:
```suggestion
The palette for the Sky effect is shared between all three sky types. To use a custom palette, you must specify all six color values as hue (0-360), saturation (0-100), brightness (0-100), and kelvin (1500-9000) in the following order to modify the effect:
```

File: source/_integrations/lifx.markdown [hunk 5]
```diff
@@ -187,6 +187,51 @@ The device will be powered on by default, but this can be overridden by setting
...
+
+1. Sky: The color of the background sky when the Clouds sky type is used. All pixels on the light are given this color, and the "clouds" are made by applying a range of saturation values to the pixels.
+2. Night sky: The starting (or finishing) color of the sky when no sun is visible for Sunrise or Sunset sky type. The Sunrise sky type starts with the light completely this color, then fades to a lighter color before starting to bring up the "sun". The Sunset sky type is the Sunrise sky type but in reverse.
+3. Dawn sky: The color of the sky just as the sun appears. This is the color that the light fades to after starting at the "night sky" color.
+4. Dawn sun: The color of the sun just as the sun appears. The background is still the "dawn sky" above, but now the sun is starting to rise with a warm color.
+5. Full sun: The color of the sun as it covers the whole light. As the sun rises, it fades from the "dawn sun" color to this "full sun" color. The background stays at "dawn sky" but does get washed out by the bright sun colors.
```
Comment (bdraco) [inline source/_integrations/lifx.markdown]:
```suggestion
5. Full sun: The color of the sun as it covers the whole light. As the sun rises, it fades from the "dawn sun" color to this "full sun" color. The background stays at "dawn sky" but gets washed out by the bright sun colors.
```

File: source/_integrations/lifx.markdown (hunk 2 above)
Comment (bdraco) [inline source/_integrations/lifx.markdown]:
```suggestion
The Flame (`lifx.effect_flame`), Morph (`lifx.effect_morph`), Sky (`lifx.effect_sky`) and Move (`lifx.effect_move`) effects are hardware-based and only work on specific LIFX devices. Flame and Morph are available on the LIFX Tile, Candle, Path, Spot, and Ceiling, while the Sky effect is only available on the Ceiling. The Move effect requires a LIFX Z, Lightstrip, Beam, Neon, or String.
```

**Review by bdraco — DISMISSED** [review]
Thanks @Djelibeybi

**Comment by Djelibeybi** [comment]
> Thanks @Djelibeybi

All good. I think I broke the AI though. Too many reviews per hour, it said. If you want it to review, it needs to be triggered in 27 minutes.

**Comment by bdraco** [comment]
@coderabbitai review

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Djelibeybi :+1:

**Pull request merged**